File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        mx = 0
        for i in nums:
            if i == 1:
                count+=1
                mx=max(count,mx)
            else:
                count=0
                
        return mx

File: Data Structures & Algorithms/binary-search/submission-0.java
class Solution {
    public int search(int[] nums, int target) {
        int len = nums.length-1;
        int start =0;


        while(start<=len) {
            int mid = start+(len-start)/2;
            if(nums[mid]==target)
                return mid;
            if(nums[mid]>target) {
                len=mid-1;
            }
            else
                start=mid+1;
        }
        return -1;
    }
}
